app/views/smart_answers/inputs/_country_select_question.html.erb

```diff
@@ -1,7 +1,9 @@
+<span class="govuk-caption-l"><%= @presenter.title %></span>
```

Comment:
Not blocking, but did we have plans to add heading-caption to the other question components?

Reply:
Hey - the other components support `  heading_caption: @presenter.title,` being passed in. Is this what you meant?

Reply:
Ah yeah - so for example the we pass the heading caption in for [checkboxes](https://github.com/alphagov/smart-answers/pull/4591/files#diff-07e4b5c3782c2696aed14e72e99b40e5R6). But here we are using a separate span for the heading, instead of passing it into the select component. Wondering are were thing about adding support for heading-caption to the select component too? If we are, I still think we can go ahead an merge this PR and update later

Reply:
Yeah - we need to review all of our components and see if they are consistent with each other. From what I've seen they are not. I will raise in the next meeting.